sdk/storage/azure-storage-queue/azure/storage/queue/_queue_client.py

```diff
@@ -683,13 +687,14 @@ def receive_messages(self, **kwargs):
             process_storage_error(error)
 
     @distributed_trace
-    def update_message(self,
-                       message,  # type: Any
-                       pop_receipt=None,  # type: Optional[str]
-                       content=None,  # type: Optional[Any]
-                       **kwargs  # type: Any
-                       ):
-        # type: (...) -> QueueMessage
+    def update_message(
+            self, message: Any,
```

Comment:
Let's update this type from `Any`. I think it's either a QueueMessage or just a message id. So new type would be `Union[str, QueueMessage]`

Confirmed in docstring below (not sure why this didn't match that.